sites/page-details/reports/pre-a11y-1.json

```diff
@@ -0,0 +1,621 @@
+{
+  "@context": "https://wet-boew.github.io/vocab/context/2023/conformance-report-en.json",
+  "@type": [
+    "earl:Assertion",
+    "acr:AssessmentReport"
+  ],
+  "earl:subject": {
+    "@id": "_:subject",
+    "dct:references": "https://example.com/web/page/url.html",
+    "dct:hasPart": [
+      {
+        "@id": "_:non-text-content_itm1",
+        "dct:isPartOf": "_:subject",
+        "@type": [
+          "earl:TestSubject",
+          "schema:WebPageElement"
+        ],
+        "dct:description": "Textual identification of the web page element that is being evaluated, like: 'Figure 2, a bar chart'. Provide a pointer when applicable (earl:pointer)",
+        "earl:pointer": {
+          "@type": "oa:CssSelector",
+          "@value": "img"
+        }
+      },
+      {
+        "@id": "_:video_abc",
+        "dct:isPartOf": "_:subject",
+        "@type": [
+          "earl:TestSubject",
+          "schema:WebPageElement"
+        ],
+        "dct:description": "Textual identification of the web page element that is being evaluated, like: 'Video ABC (earl:pointer)"
+      }
+    ],
+    "@type": [
+      "earl:TestSubject",
+      "schema:WebPage"
+    ],
+    "dct:description": "Describe what is being assessed, like the full page or a component inside that page. Provide a pointer when applicable (earl:pointer)",
+    "earl:pointer": {
+      "@type": "oa:CssSelector",
+      "@value": "title, meta, #wb-bc, main"
+    }
```

Comment:
I think the test subject are not well defined.

If this assessment is limited to the page-details, then the `earl:pointer` value of the main subject should be a CSS selector that only select the page-details section inside the main area.

Is the page-details have sub-sections where it contains images and video? If there is no sub-section, the JSON object "dct:hasPart" should be removed.

The test page subject URL need to be more specific.